README.md

```diff
@@ -23,10 +48,6 @@ Afterwards you can install this node via
 cargo install canvas-node --git https://github.com/paritytech/canvas-node.git --force
```

Comment:
Since this section is called "Building from source" I'd recommend changing this command to something like: `git clone ... && cd repo && cargo build --release`